reInvent18_Developer_Workshop/helloworld_c/README.md

```diff
@@ -31,8 +31,10 @@ where examples is the name of the directory where the repository will be stored
 ## 3. SOFTWARE AND SYSTEM REQUIREMENTS
 Board | Device Name | Software Version
 ------|-------------|-----------------
```

Comment:
Why are we referencing vcu and kcu devices?

Reply:
This README is from xilinx SDAccel Examples directory.  Since this was in the previous version of this workshop i copied over the latest version. However this README is not used or referenced during the workshop and hence is not necessary. I will remove this README file.